Source/JavaScriptCore/interpreter/Interpreter.cpp

```diff
@@ -527,9 +527,12 @@ void Interpreter::getStackTrace(JSCell* owner, Vector<StackFrame>& results, size
                 break;
             }
             }
-        } else if (!!visitor->codeBlock() && !visitor->codeBlock()->unlinkedCodeBlock()->isBuiltinFunction())
-            results.append(StackFrame(vm, owner, visitor->callee().asCell(), visitor->codeBlock(), visitor->bytecodeIndex()));
-        else
+        } else if (!!visitor->codeBlock() && !visitor->codeBlock()->unlinkedCodeBlock()->isBuiltinFunction()) {
+            // We should skip async function wrapper frame, only append body frame for async function
```

Comment:
Why not adding implementation visibility flag to the wrapper?

Reply:
Sorry, I spent a day investigating for a parser-side fix but it was a difficult. At least currently, it seems designed so that the wrapper and body have the same implementationVisibility set.

Is the current implementation of this PR acceptable with a FIXME comment?